### PR TITLE
Add info to use MailDev to view emails

### PR DESCRIPTION
**Pull request by vbudhram**
This PR adds some documentation and server configuration to use fxa-local-dev with MailDev. This will allow developers/users the ability to inspect the emails being sent.

@vladikoff Do you think this is worthwhile? r?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -228,3 +228,24 @@ nvm exec 0.10 ./pm2 start servers_extra.json
...
+If you want to inspect emails, you can run fxa-local-dev with MailDev.
+
+#### Install
+```bash
+npm install maildev -g
+```
+
+#### Run
+```bash
+./pm2 start servers_maildev.json
```
Comment (TDA) [inline README.md]: Is this really required? ./pm2 start servers.json works fine, and we already have that documented.

Comment (vladikoff) [inline README.md]: @vbudhram yeah probably avoid a new maildev json, just say stop `./pm2 stop 0` ?

Comment (vbudhram) [inline README.md]: Sounds good!

**Comment by vladikoff** [comment]
> @vladikoff Do you think this is worthwhile? r?


Yeah i think so, this would be really useful!

File: README.md [hunk 2]
```diff
@@ -228,3 +228,24 @@ nvm exec 0.10 ./pm2 start servers_extra.json
 
 #### Run
 Once services have started, run `npm start` to open Firefox with a local profile. Access the Hello service as you normally would.
+
+### Running with MailDev
+
+If you want to inspect emails, you can run fxa-local-dev with MailDev.
```
Comment (vladikoff) [inline README.md]: @vbudhram can we expand why would this be useful?

Comment (vladikoff) [inline README.md]: Also you can link `MailDev` to https://www.npmjs.com/package/maildev or github

File: README.md [hunk 3]
```diff
@@ -228,3 +228,24 @@ nvm exec 0.10 ./pm2 start servers_extra.json
 
 #### Run
 Once services have started, run `npm start` to open Firefox with a local profile. Access the Hello service as you normally would.
+
+### Running with MailDev
```
Comment (vladikoff) [inline README.md]: @vbudhram could you add a ` *******` above the heading to separate from FF Hello

**Comment by vladikoff** [comment]
@vbudhram excellent, thanks!!

**Pull request merged**